docs/getting-started/kubernetes-quickstart.md

```diff
@@ -0,0 +1,155 @@
+
+NAMESPACE     NAME                               READY   STATUS    RESTARTS   AGE 
+kube-system   coredns-565d847f94-kd9xx           1/1     Running   0          15s 
+kube-system   etcd-minikube                      1/1     Running   0          26s 
+kube-system   kube-apiserver-minikube            1/1     Running   0          26s 
+kube-system   kube-controller-manager-minikube   1/1     Running   0          26s 
+kube-system   kube-proxy-cvqjm                   1/1     Running   0          15s 
+kube-system   kube-scheduler-minikube            1/1     Running   0          26s 
+kube-system   storage-provisioner                1/1     Running   0          15s 
+```
+
+## Running tracee as a daemon set
+
+Tracee uses eBPF for host-based detections, meaning it has visibility into all processes running on a given host, regardless of them being separated into different containers or pods. Therefore, we want to have tracee running on each node.
+
+The following kubernetes manifest can be used to run tracee as a daemon set. This tells Kubernetes to have a single tracee process on each node.
+
+```
+--- 
+apiVersion: apps/v1 
+kind: DaemonSet 
+metadata: 
+  labels: 
+    app.kubernetes.io/name: tracee 
+    app.kubernetes.io/component: tracee 
+    app.kubernetes.io/part-of: tracee 
+  name: tracee 
+spec: 
+  selector: 
+    matchLabels: 
+      app.kubernetes.io/name: tracee 
+  template: 
+    metadata: 
+      labels: 
+        app.kubernetes.io/name: tracee 
+      name: tracee 
+    spec: 
+      containers: 
+      - name: tracee 
+        image: docker.io/aquasec/tracee:0.8.3 
+        imagePullPolicy: IfNotPresent 
+        env: 
+          - name: LIBBPFGO_OSRELEASE_FILE 
+            value: /etc/os-release-host 
+        securityContext: 
+          privileged: true 
+        volumeMounts: 
+        - name: tmp-tracee 
+          mountPath: /tmp/tracee 
+        - name: etc-os-release 
+          mountPath: /etc/os-release-host 
+          readOnly: true 
+        # NOTE: Resource consumption will vary between different use cases and 
+        # workload characteristics. User should monitor tracee for resource 
+        # consumption before enabling resource limits. Capping tracee 
+        # resources may cause loss of events and miss detections. 
+        # resources: 
+        #   limits: 
+        #     cpu: "1" 
+        #     memory: 1Gi # tracee has a 512MB in-memory events cache enabled by default 
+        #   requests: 
+        #     cpu: "1" 
+        #     memory: 1Gi 
+      tolerations: 
+        - effect: NoSchedule 
+          operator: Exists 
+        - effect: NoExecute 
+          operator: Exists 
+      volumes: 
+      - hostPath: 
+          path: /tmp/tracee 
+        name: tmp-tracee 
+      - hostPath: 
+          path: /etc/os-release 
+        name: etc-os-release 
+```
+
+After saving the above manifest into a file called `tracee.yml` you can install it via: 
+
+```
+[*] kubectl apply –f tracee.yml 
+```
```

Comment:
```suggestion
After saving the above manifest into a file called `tracee.yml` you can install it via: 

```
[*] kubectl apply –f tracee.yml 
```
```

Let's use the one directly from tracee repo: 

```
kubectl apply -f https://github.com/aquasecurity/tracee/blob/main/deploy/kubernetes/tracee/tracee.yaml
```